docs/config.adoc

```diff
@@ -462,6 +462,11 @@ The default length limit is 128 keys.
 === Linux only: linux-dev
 <<table-of-contents,Back to ToC>>
 
+NOTE: The format for specifying multiple Linux devices has changed in v1.5.0 to a new one, based on lists.
+The old format is no longer presented in this document, but it will still work for compatibility reasons. To see the old specification
+https://github.com/jtroo/kanata/blob/v1.5.0-prerelease-3/docs/config.adoc#linux-only-linux-dev[click here].
```

Comment:
I'm not sure, if this should be hidden behind a link like this or the new list format added as alternative info added next to already existing info about. 

I'm beginning to think the second option is better, otherwise it might cause more problems than it solves.

Reply:
The old way won't be deprecated, and agreed that a link is less preferable.